src/util-dpdk.c

```diff
@@ -18,16 +18,22 @@
 /**
  * \file
  *
- * \author Lukas Sismis <lukas.sismis@gmail.com>
+ * \author Lukas Sismis <sismis@cesnet.cz>
  */
 
+#ifndef UTIL_DPDK_C
+#define UTIL_DPDK_C
+
 #include "suricata.h"
 #include "util-dpdk.h"
 
+// equal to -1 to get the first lcore id
+int dpdk_last_spawned_lcore = -1;
```

Comment:
how/why is this global used?

Reply:
The global variable was used to store the last lcore onto which a Suricata worker/management thread had been spawned. However, as I am only spawning workers in the current PR, I've moved the variable definition inside of the function `RunModeSetLiveCaptureWorkersForDevice`.